openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -178,4 +179,4 @@ def patch(self, request, username):
                 status=status.HTTP_400_BAD_REQUEST
             )
 
-        return Response(status=status.HTTP_204_NO_CONTENT)
+        return Response(account_settings)
```

Comment:
@catong FYI. This PR changes the return value from 204 (with no content) to 200 (with the updated data) for the `AccountView` PATCH method.